packages/core/src/controllers/ApiController.ts

```diff
@@ -61,8 +61,8 @@ export const ApiController = {
 
     return {
       'x-project-id': projectId,
-      'x-sdk-type': sdkType,
-      'x-sdk-version': sdkVersion
+      'x-sdk-type': sdkType || 'w3m',
```

Comment:
Remove these default values, and make sure the sdkType and sdkVersion values are set eventually